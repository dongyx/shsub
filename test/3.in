<%	name=shsub
	desc="substitute embedded shell commands and expressions with \
their outputs and results"
%><%=$name%> <%=$desc%>.
Best wish.
