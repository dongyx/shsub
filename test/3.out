shsub substitute embedded shell commands and expressions with their outputs and results.
Best wish.
